### ISSUE TITLE
Navy Federal Credit Union

### ISSUE THREAD

**Issue by rob6185** [n00neimp0rtant/xCon-Issues#494]
the app version 4.3 is able to detect jail breaking. It's a hassle always typing in my acct info and would be better just using the new Touch ID they just implemented.

**Comment by xtekfirex** [comment]
I was just about post the same issue! I'm about to switch banks!

**Comment by psedog** [comment]
With Xcon the NavyFed app is working fine for me. I've tried 40 beta 1 and beta 5 and the app opens up no problem. Even if it didn't it wouldn't make me leave the bank. I would just go back to using the website. The website doesn't care either way.

**Comment by rob6185** [comment]
Yes but even with Xcon the NavyFed app recognizes that the phone is jail broken and won't allow the Touch ID or remember me to be enabled. I backtracked to the previous version so I could still use the remember me function and just input my pin.  With the newest update, while it sees the JB status, I have to put in access code and password every time. 

Sent from my AT&T iPhone 6.
"Can you hear me now? No."

**Comment by psedog** [comment]
I'm using Touch ID as well. I wouldn't have commented if it wasn't working. 

http://psedog.com/hostedfiles/iPhone/IMG_3748.PNG

**Comment by rob6185** [comment]
Hmm I don't know why it's telling me it can't. Maybe other jailbreak apps are causing it?

**Comment by psedog** [comment]
If you have My3G, 3G unrestrictor, or another app like that I would look there.

**Comment by TheLlamaAbides** [comment]
Latest v40 beta fixes NFCU app with TouchID working fine, but then Evernote pops cocoa error 513 when trying to enter local updates.
